app/models/miq_server/worker_management/monitor.rb

```diff
@@ -63,6 +66,21 @@ def sync_workers
     result
   end
 
+  def cleanup_failed_workers
+    if podified?
+    elsif systemd?
+      cleaup_failed_systemd_services
```

Comment:
typo ^

Reply:
:+1: added a spec to worker_management/monitor_spec also